### ISSUE TITLE
content not displaying

### ISSUE THREAD

**Issue by robbyking3** [SC5/mozaik-ext-sheets#6]
Hi! Firstly thanks for making this extension, open source things are awesome.

I'm trying to use this extension. I've signed up for the Google drive api, gotten the p12 file, converted it to .pem with openssl, and reference it through the .env value, along with the google service email. 

Node was complaining about reflux so I manually added it as a dev dependency to package.json, and the error went away.

In the `src/App.jsx` I import sheets.

```
import React   from 'react';
import Mozaik  from 'mozaik/browser';
import github  from 'mozaik-ext-github';
import travis  from 'mozaik-ext-travis';
import time    from 'mozaik-ext-time';
import sheets  from 'mozaik-ext-sheets';

const MozaikComponent = Mozaik.Component.Mozaik;
const ConfigActions   = Mozaik.Actions.Config;


Mozaik.Registry.addExtensions({
    github,
    travis,
    sheets,
    time
});

React.render(<MozaikComponent/>, document.getElementById('mozaik'));
```

and in `src/server.js`

```
import Mozaik  from 'mozaik';
import config  from '../config';
import github  from 'mozaik-ext-github/client';
import travis  from 'mozaik-ext-travis/client';
import sheets  from 'mozaik-ext-sheets/client';

const mozaik = new Mozaik(config);

mozaik.bus.registerApi('sheets', sheets);
mozaik.bus.registerApi('github',  github);
mozaik.bus.registerApi('travis',  travis);

mozaik.startServer();
```

I run `npm run build-assets` just in case, then do `npm start`

and on the browser there is a console error 
`Uncaught TypeError: Cannot read property 'ApiConsumer' of undefined`

from a line in List.jsx `var ApiConsumerMixin = require("mozaik/browser").Mixin.ApiConsumer;`

So I tried changing it to `var ApiConsumerMixin = require("mozaik/browser").default.Mixin.ApiConsumer;` and it loads the page and shows how many rows have loaded, but none of the content actually displays. It just creates as many list items as are rows, without content. 

any ideas?

**Comment by robbyking3** [comment]
I tried adding the fields parameter and just outputting `'{A}', '{B}'` 
the content then displays. I'm gonna fork this and see if I can fix it so fields can be empty by default and show all content.

**Comment by juhamust** [comment]
Thanks for reporting the issue. I think the module needs some dependency and code updates in order to work with Mozaik 1.4.x. I'm also going to update the codebase into ES6 while at it.

**Comment by juhamust** [comment]
Fixed in release 0.4.0 (2bcd995d276b)
